code/modules/mob/dead/observer/observer.dm

```diff
@@ -465,7 +465,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(config.disable_player_mice)
 		src << "<span class='warning'>Spawning as a mouse is currently disabled.</span>"
 		return
-

```

Comment:
Not the removal of the space, which I just went back and added, but this:
	if(ticker.current_state < GAME_STATE_PLAYING)
		src << "<span class='warning'>You can not spawn as a mouse before round start!</span>"
		return